src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationSettingTests.cs

```diff
@@ -50,10 +51,11 @@ public void FilterContains()
                 Labels = new List<string>() { "*label*" },
             };
 
-            var builder = new UriBuilder();
+            var builder = new HttpPipelineUriBuilder();
+            builder.Uri = new Uri("http://localhost/");
             service.BuildBatchQuery(builder, selector);
 
-            Assert.AreEqual(builder.Uri.AbsoluteUri, "http://localhost/?key=*key*&label=*label*");
+            Assert.AreEqual("http://localhost/?key=*key*&label=*label*", builder.Uri.AbsoluteUri);
```

Comment:
out of curiosity, does the order matters?

Reply:
Yes, when NUnit formats the error message one is titled "expected" the other "actual". Expected always goes first in all Asserts.

Reply:
Thanks for the clarification! I didn't know :)